### ISSUE TITLE
Description need

### ISSUE THREAD

**Issue by QuAzI** [Wolfsblvt/BlossomsPokemonGoManager#298]
Please make some documentation ( Menu -> Help or Markdown document ) with description of columns, how it calculated and how user can use it.
Now I don't understand half of new columns. I think not only I am.

**Comment by Wolfsblvt** [comment]
Yeah, you are totally right. Some columns are complicated and not explained well.
What I do plan for this is this: With the new UI, columns will be hideable and hidden can be added. There will be a menu where each possible column is listed, with it's name and a detailed explanation what this column means.
Then it's even in the tool.

**Comment by AbsoluteDestiny** [comment]
Honestly even just a wiki entry or a link to a page where these terms are described would be useful.

**Comment by Wolfsblvt** [comment]
We are working on a short wiki article soon.
Guess I will do it when I have finished working on the current columns.

**Comment by GreeKatrina** [comment]
Maybe to help get this rolling, my guess as to what the column descriptions are (for ones that may be confusing):

| Name | Description |
| --- | --- |
| Max CP (Cur) | Maximum CP of pokemon based on user's current level. |
| Max CP (40) | Maximum CP of pokemon if user was at level 40. (Max CP does not increase after lvl 40.) |
| Max Evolved CP (Cur) | Maximum CP pokemon can obtain if evolved. Same as Max CP (Cur) if last evolution. |
| Max Evolved CP (40) | Maximum CP pokemon can obtain if evolved and user was at level 40. Same as Max CP (40) if last evolution. |
| Duel Ability | Relative gym offensive power assuming the pokemon just takes incoming damage and not doing any dodging. |
| Gym Offense | Relative damage (0 - 100) the pokemon can inflict while battling (using the better value between just using quicks vs weaving) multiplied by the Base Atk value of the species. |
| Gym Defense | Relative ability (0 - 100) the pokemon has to defend a gym. |
| CP Evolved | Estimate of the pokemon's CP after evolving. |
| Evolvable | Number of pokemon you can evolve with the current amount of candies you have for that species. |

Duel Ability IV, Gym Offense IV, and Gym Defense IV are the relative ratings of the pokemon from the corresponding column names based on their IV%.

I'm guessing Duel Ability, Gym Offense, and Gym Defense are the same definitions from [here](https://www.reddit.com/r/TheSilphRoad/comments/4vcobt/posthotfix_pokemon_go_full_moveset_rankings/).

**Comment by Wolfsblvt** [comment]
Thanks for reporting this and for using BlossomsPokemonGoManager. This project is no longer maintained and no longer works with current Pokémon Go services, so I’m closing all remaining issues as part of archiving it. Please do not use it with a live account or current credentials. See the [Disclaimer](https://github.com/Wolfsblvt/BlossomsPokemonGoManager#disclaimer) for details.

**Comment by Wolfsblvt** [comment]
Thanks for reporting this and for using the project. Blossoms Pokémon Go Manager is no longer maintained and no longer works with current Pokémon Go services, so I’m closing all remaining issues as part of archiving the repository. This avoids leaving a false impression that further development or support is planned.
